### ISSUE TITLE
error when generating mgmt-plane web service

### ISSUE THREAD

**Issue by weidongxu-microsoft** [Azure/autorest.modelerfour#281]
**Describe the issue**
FATAL error when generating mgmt-plane web service:
```
FATAL: YAMLException: bad indentation of a sequence entry in "prechecked-openapi-document.yaml" at line 3436, column 25:
                            value: auditlogs
                            ^
  Error: Plugin modelerfour reported failure.
```

Cause is example file

https://github.com/Azure/azure-rest-api-specs/blob/master/specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/Diagnostics.json#L588

https://github.com/Azure/azure-rest-api-specs/blob/master/specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/examples/Diagnostics_ExecuteSiteAnalysisSlot.json#L34

Same as https://github.com/Azure/autorest.modelerfour/issues/159 (I cannot reopen it), so the new issue.

** REQUIRED ** 
**ModelerFour** version: 4.13.348
**AutoRest-core** version: 3.0.6282
**CommandLine** : 3.0.6187
OpenAPI/Swagger file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/Diagnostics.json

**Comment by daviwil** [comment]
I'm pretty confident that PR https://github.com/Azure/perks/pull/105 fixes this issue, it's now been released in `@autorest/modelerfour` [4.13.351](https://github.com/Azure/autorest.modelerfour/releases/tag/v4.13.351).  Please try it out and let me know if it works for you!

**Comment by weidongxu-microsoft** [comment]
@daviwil 

Thanks David! It is good now on 4.13.351.
